ClientSideCode/Carian/__tests__/screen/Departmentpage-test.js

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import DepartmentPage from '../../screen/drawerScreens/DepartmentPage';
+import { View, Text, TextInput, TouchableOpacity,ScrollView } from 'react-native';
+import { expect } from 'chai';
+import sinon from 'sinon';
+import styles from '../../styles/commonStyles';
+import RadioForm from 'react-native-simple-radio-button';
+const navigation = { navigate: jest.fn() };
+global.expect = expect;
+global.sinon = sinon;
+global.shallow = shallow;
+
+describe('<DepartmentPage/>', () => {
+  beforeEach(function () {
+    spyon = sinon.spy(navigation, 'navigate');
+    wrapper = shallow(<DepartmentPage navigation={navigation}></DepartmentPage>);
+  });
+
+  afterEach(function () {
+    navigation.navigate.restore();
+  });
+  it('should have Scrollview', () => {
+    expect(wrapper.type()).to.equal(ScrollView);
+  });
+  it('should have View', () => {
+    expect(wrapper.find(View)).to.have.length(6);
+  });
+  it('should have Radioform', () => {
+    expect(wrapper.find(RadioForm)).to.have.length(1);
+  });
+  it('should have TextInput', () => {
+    expect(wrapper.find(TextInput)).to.have.length(4);
+  });
```

Comment:
you can write more detailed test cases of text boxes. refer to doctor public profile or hospital public profile